ydb/public/api/protos/draft/ymq.proto

```diff
@@ -0,0 +1,271 @@
+syntax = "proto3";
+
+import "ydb/public/api/protos/ydb_operation.proto";
+
+import "google/protobuf/descriptor.proto";
+
+package Ydb.YMQ.V0;
+option java_package = "com.yandex.ydb.ymq.v0";
+
+// TODO: move to common place with DataStreams
+enum EFieldTransformationType {
```

Comment:
Can we use this only once? This is copy-paste from datasstreams.proto.

Reply:
Moved it to common proto file